### ISSUE TITLE
Cover the project with tests(not a bug)

### ISSUE THREAD

**Issue by jcbritobr** [AllenDang/giu#219]
First of all, i want to congratulates by the very nice project. It is a very important step for a really effective golang ui toolkit. To make it solid, it only needs a good test coverage layer to make it perfect.

**Comment by AllenDang** [comment]
@jcbritobr Sure thing, I'm doing it right now.

**Comment by gucio321** [comment]
@AllenDang idk if i already asked about it (iitc i did but don't remember and can't find where):
Do you have any idea about how to write unit Tests for imgui widgets?

**Comment by jcbritobr** [comment]
Mock native data in unit tests. Objective is not test imgui software, but only your wrapper.

**Comment by gucio321** [comment]
Yeah you are right. I'd need a code example and then i can do that 😊
